### ISSUE TITLE
Export new datasource

### ISSUE THREAD

**Issue by Gabbosaur** [dabeng/react-orgchart#28]
Hi, is there a way to export the datasource upon adding/deleting/updating the chart? Thanks.

**Comment by DEADushka108** [comment]
you can use this variant https://github.com/dabeng/react-orgchart/blob/master/src/edit-chart/edit-chart.js . In this example you can use JSONDigger and useState for all manipulations with your ds.

**Comment by jnachman123** [comment]
is there anyway to call the orgchart and get the current datasource status. e.g. console.log(orgchart.datasource)

**Comment by Gabbosaur** [comment]
> is there anyway to call the orgchart and get the current datasource status. e.g. console.log(orgchart.datasource)

I managed to do it with this.

```
  const getHierarchy = async () => {
     const hierarchy = await dsDigger.findNodeById("{ROOT-NODE-ID}");
     console.log(JSON.stringify(hierarchy));
   };
```

where ROOT-NODE-ID is the id of your root's node. Then you can call the function getHierarchy anywhere you prefer and it prints out the datasource.

**Comment by jnachman123** [comment]
@Gabbosaur   thanks , forgot to mention that i am using react. and i understand thats not available in that version?

**Comment by Gabbosaur** [comment]
> @Gabbosaur thanks , forgot to mention that i am using react. and i understand thats not available in that version?

That's right. You can use the function I previously posted to get the datasource, and yes I also have used React.
Hope that helps!
